### PR TITLE
feat: clean code

### PR DESCRIPTION
**Pull request by ririxidev**
This pull request includes:
  - [fix(extensions): remove ts-checks](https://github.com/spicetify/spicetify-cli/commit/42d8ac7f3a957e0785ab36ae61f1f64856f7e594)
  - [chore(github/actions): depend build on 1.19](https://github.com/spicetify/spicetify-cli/commit/b93c0bc88c6f5b69a3c06754393a6bb5277189f2)
  - [style(prettier): modify config](https://github.com/spicetify/spicetify-cli/commit/224b0df758f14bdadb644c557673aa50907c059a)
  - [feat(github/actions): make prettier write to pr](https://github.com/spicetify/spicetify-cli/commit/f4a49c583d4c237b898e66049ce39f5a3df510cf)
    - This GitHub Action hasn't been tested
  - [style: run prettier on files](https://github.com/spicetify/spicetify-cli/commit/7dc3611d666d77e126e64ac581a880d0f9a2bce5)
  - [feat(golang): upgrade to 1.19](https://github.com/spicetify/spicetify-cli/pull/1992/commits/858bc50e0844586027ba82033ed3a061c406911b)
    - Builds on my machine:
       - macOS (`Darwin *redacted* 21.6.0 Darwin Kernel Version 21.6.0: Mon Aug 22 20:17:10 PDT 2022; root:xnu-8020.140.49~2/RELEASE_X86_64 x86_64`) Monterey (12.6)
       - go `version go1.19.2 darwin/amd64`

### CODE REVIEW

**Comment by kyrie25** [comment]
If we're planning to upgrade build action to Go `1.19` then we should also probably make changes to `go.mod` accordingly?

**Review by kyrie25 — APPROVED** [review]
Looks all good to me, we should also refactor and remove the deprecated `io/ioutil` that's been whining since Go `1.16` lol, but that's a sidenote anyway.
I'll leave the code style config for the team to decide

**Comment by ririxidev** [comment]
Prettier config has been discussed months ago with @afonsojramos (even is in old branch with the same changes) but I never made pull request with these changes

**Review by afonsojramos — COMMENTED** [review]
On board with everything except the removal of the ts-checks. We know that some errors arise due to this, but overall I see more benefit over having the checks than not.

**Comment by ririxidev** [comment]
> On board with everything except the removal of the ts-checks. We know that some errors arise due to this, but overall I see more benefit over having the checks than not.

I think we already discussed it few months ago, and we decided to remove them for the sake. TypeScript checks should be in ts files and not in javascript ones.

**Comment by afonsojramos** [comment]
> > On board with everything except the removal of the ts-checks. We know that some errors arise due to this, but overall I see more benefit over having the checks than not.
> 
> I think we already discussed it few months ago, and we decided to remove them for the sake. TypeScript checks should be in ts files and not in javascript ones.

Well, if I could change to TS I would, but obviously that here it is rather unfeasible. But sure, if everyone agrees I don't want to be the odd one out.

**Review by afonsojramos — APPROVED** [review]

File: .github/workflows/prettier.yml [hunk 1]
```diff
@@ -1,14 +1,21 @@
 name: Prettier
 
-on: [pull_request, push]
+on: [pull_request]
 
 jobs:
-  prettier:
+  prettierPR:
```
Comment (afonsojramos) [inline .github/workflows/prettier.yml:6]:
```suggestion
  prettier:
```

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**